lib/studies/rally/login.dart

```diff
@@ -394,11 +393,15 @@ class _FilledButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return FlatButton(
-      color: RallyColors.buttonColor,
-      padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 24),
-      shape: RoundedRectangleBorder(
-        borderRadius: BorderRadius.circular(12),
+    return ElevatedButton(
+      style: ElevatedButton.styleFrom(
+        primary: RallyColors.buttonColor,
+        onPrimary: Colors.black,
+        elevation: 0,
```

Comment:
Is there some other combination in the new universe that achieves the same result as an "ElevatedButton with 0 elevation" ?

Reply:
Yes, using a text button with a background color will achieve this too, and avoids adding elevation on press/hover, so I'll update it